wbooks/android/settings.gradle

```diff
@@ -1,3 +1,5 @@
 rootProject.name = 'wbooks'
+include ':react-native-config'
+project(':react-native-config').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-config/android')
```

Comment:
No, autolink is deprecated for RN > 0.60. Be careful with this!

Reply:
![image](https://user-images.githubusercontent.com/45558924/106556094-9ab52b80-64fd-11eb-9cc9-0dbd289b6fed.png)

But this is listed on the repository readme, listed under manual link.

Reply:
We are not using the manual link, RN has the autolink feature for 0.60+ version